webhooks/accounts.go

```diff
@@ -0,0 +1,27 @@
+package webhooks
+
+import "encoding/xml"
+
+// Account notifications.
+// https://dev.recurly.com/page/webhooks#account-notifications
+const (
+	BillingInfoUpdated = "billing_info_updated_notification"
+)
+
+// AccountNotification is returned for all account notifications.
+type AccountNotification struct {
+	Type    string  `xml:"-"`
+	Account Account `xml:"account"`
+}
+
+// Account represents the account object sent in webhooks.
+type Account struct {
+	XMLName     xml.Name `xml:"account"`
+	Code        string   `xml:"account_code,omitempty"`
```

Comment:
We should be able to remove `omitempty` from all the struct tags on notifications since these are used for reads only.